### PR TITLE
Provided encoding for input files to ingestor via mapping

### PR DESCRIPTION
**Pull request by DanielHorbunov**
Modified _ _ init _ _, opener methods in AbstractIngestor class:
_ _ init _ _: added input_encoding field, which used when reading input files in a specific encoding
opener: provided encoding argument in smart_open.open

### CODE REVIEW

File: thebeast/ingest/abstract.py [hunk 1]
```diff
@@ -17,8 +17,9 @@ class AbstractIngestor:
     more params (for example, credentials for a remote source)
     """
 
-    def __init__(self, input_uri: str, *args, **kwargs) -> None:
+    def __init__(self, input_uri: str, input_encoding: str, *args, **kwargs) -> None:
```
Comment (dchaplinsky) [inline thebeast/ingest/abstract.py:20]: Add an utf-8 default on next commit

File: beast.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import sys
 
 if __name__ == "__main__":
-    mapping = SourceMapping(Path("thebeast/tests/sample/mappings/ukrainian_mps.yaml"))
```
Comment (dchaplinsky) [inline beast.py:7]: Better keep one test mapping for now.

File: thebeast/ingest/abstract.py (hunk 1 above)
Comment (dchaplinsky) [reply thebeast/ingest/abstract.py:20]: Keep in mind that some sources might not be files (but APIs, dbs, etc)

Comment (DanielHorbunov) [reply thebeast/ingest/abstract.py]: Added an utf-8 default. I will think about other sources later

**Pull request merged**